Gemfile

```diff
@@ -57,6 +57,7 @@ gem "lograge"
 gem "logstash-event"
 
 ## custom
+gem "bulk_email", "~> 0.0.1", path: "vendor/engines/bulk_email"
```

Comment:
Theoretically since we're just pointing to a path, we don't really need the version here. We could conceivably remove all of the version strings until they get extracted into an action gem.